crates/uv/src/cli.rs

```diff
@@ -1703,14 +1703,14 @@ pub(crate) struct VenvArgs {
     #[arg(long)]
     pub(crate) seed: bool,
 
-    /// Overwrite the directory at the specified path when creating the virtual environment.
+    /// Preserve any existing files or directories at the target path.
     ///
     /// By default, `uv venv` will remove an existing virtual environment at the given path, and
-    /// exit with an error if the path is non-empty but _not_ a virtual environment. The `--force`
-    /// option will instead write to the given path, regardless of its contents, and without
-    /// clearing it beforehand.
+    /// exit with an error if the path is non-empty but _not_ a virtual environment. The
+    /// `--allow-existing` option will instead write to the given path, regardless of its contents,
```

Comment:
Should we say like `--unsafe-allow-existing` since it could bork the environment? We could make it a safe flag in the future with checks for compatible interpreter versions?

Reply:
Do we need to document the caveats of how this could break an environment?

Reply:
I'll document it. It should _generally_ be fine since the virtual environment at least stores packages under their own minor Python version? virtualenv already does this today as its default behavior.